build.sh

```diff
@@ -801,6 +801,39 @@ case "$COMMAND" in
         ;;
 
     ######################################
+    # Continuous Integration
+    ######################################
+
+    "ci-pr")
+        mkdir -p build/reports
+
+        if [ "$target" = "docs" ]; then
+            ./build.sh set-swift-version
+            ./build.sh verify-docs
+        elif [ "$target" = "swiftlint" ]; then
+            ./build.sh verify-swiftlint
+        else
+            sha=$ghprbSourceBranch
+            REALM_SWIFT_VERSION=$swift_version
```

Comment:
These need to be exported to actually do anything as they're used from child processes.